### PR TITLE
sellauto and buyauto hooks

### PR DESCRIPTION
**Pull request by lututui**

### CODE REVIEW

**Comment by oleokoong** [comment]
Thanks for the hook~

**Comment by oleokoong** [comment]
@Nipodemos 
can help me check these? 
because my get error..

automacro clearout {
	SimpleHookEvent AI_buy_auto_done
	timeout 30
	exclusive 1
	call clearoutM
}
macro clearoutM {
        do respawn
}
 [eventMacro] Event of type 'hook', and of name 'AI_buy_auto_done' activated automacro 'clearout', calling macro 'clearoutM '
 [eventMacro] Running macro 'clearoutM' got orphaned, its orphan method is 'terminate'.

**Comment by sctnightcore** [comment]
```
automacro clearout {
SimpleHookEvent AI_buy_auto_done
      call {
	     wait 30
	     do respawn
	    }
}	
```

File: src/AI/CoreLogic.pm [hunk 1]
```diff
@@ -1682,6 +1682,7 @@ sub processAutoSell {
 
 			$args->{sentSellPacket_time} = time;
 			
+			Plugins::callHook('AI_sell_auto_complete');
```
[the remark this answers is not in the document]
Comment (lututui) [reply src/AI/CoreLogic.pm]: Oh wow brainfart

**Pull request merged**